packages/strapi-design-system/src/Button/Button.js

```diff
@@ -1,5 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { Text } from '../Text';
+import { Box } from '../Box';
+import { getDisabledStyle, getHoverStyle, getActiveStyle, getVariantStyle, getBoxPosition } from './utils';
+import { VARIANTS, BUTTON_SIZES } from './constants';
 
-export const Button = styled.button`
-  background: ${(props) => props.theme.colors.primary100};
+export const ButtonStyle = styled.button`
+  display: flex;
+  cursor: pointer;
+  padding: ${({ theme }) => `${theme.spaces[2]} ${theme.spaces[4]}`};
+  border-radius: ${({ theme }) => theme.borderRadius};
+  background: ${({ theme }) => theme.colors.primary600};
+  border: none;
+  ${Box} {
+    display: flex;
+    align-items: center;
+    margin-top: ${getBoxPosition};
+  }
+  ${Text} {
+    color: ${({ theme }) => theme.colors.neutral0};
+  }
+  svg {
+    height: ${({ theme }) => theme.spaces[3]};
+  }
+  svg {
+    > g,
+    path {
+      fill: ${({ theme }) => theme.colors.neutral0};
+    }
+  }
+  &:disabled {
```

Comment:
I'm not 100% sure how to deal with the disabled state of things when dealing with accessibility. Because it prevents people from even knowing that there is a button (remember the submit form example?). But for today, I don't have strong appreciations :(